src/Sylius/Behat/Behaviour/WaitsForElements.php

```
@@ -0,0 +1,121 @@
+
+    protected function waitForPageToLoad(): void
+    {
+        $result = $this
+            ->getSession()
+            ->wait(5000, "document.readyState === 'complete' && !document.querySelector('[data-live-is-loading]')")
+        ;
+
+
+        // If timeout, log current state for debugging
+        if (!$result) {
+            $status = $this->getSession()->evaluateScript(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$status'. (undefined)

(UnusedLocalVariable)